docs/factory_value_object.md

```diff
@@ -18,6 +18,13 @@ manner. The value passed here can be any of the following:
 * Finally, the value can be a closure. In this case, the form data is passed as arguments to this anonymous function
   allowing for full flexibility on how to create the initial object.
 
+The `factory` and `data_class`-option are mutually exclusive, meaning you cannot set them both at the same time. If you
```

Comment:
It's not the `factory` option that forbids it, but the `immutable` option, right?